resources/windows/wix/componentrefs.wxs.erb

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+  <Fragment>
+    <ComponentGroup Id="ServiceListGroup">
+      <%- get_services.each do |service| -%>
+      <ComponentGroupRef Id="Service_<%= service.id %>" />
```

Comment:
This should be just ComponentRef, not ComponentGroupRef, since the service itself is not a component group, just a single component